solutions/standard/override.json

```diff
@@ -0,0 +1,1070 @@
+                        }
+            },
+	    {
+                  "direction": "outbound",
+                  "name": "allow-all-outbound",
+                  "source": "0.0.0.0/0",
+                  "tcp": {
+                            "port_max": 22,
+                            "port_min": 22
+                        }
+           },
+	   {
+                  "direction": "outbound",
+                  "name": "allow-ibm-outbound-1",
+                  "source": "0.0.0.0/0",
+                  "icmp": {
+                            "type": 8,
+			    "code": 0
+                        }
+            },
+	    {
+                  "direction": "outbound",
+                  "name": "allow-all-outbound-2",
+                  "source": "0.0.0.0/0",
+                  "tcp": {
+                            "port_max": 992,
+                            "port_min": 992
+                        }
+           },
+	   {
+                  "direction": "outbound",
+                  "name": "allow-all-outbound-3",
+                  "source": "0.0.0.0/0",
+                  "tcp": {
+                            "port_max": 10443,
+                            "port_min": 10443
+                        }
+           }
+ 	  ]
+          },
+	  {
+            "name": "site-to-site-sg",
+            "vpc_name": "workload",
+	     "resource_group": "slz-workload-rg",
+            "rules": [
+                  {
+                  "direction": "inbound",
+                  "name": "allow-ibm-inbound",
+                  "source": "0.0.0.0/0",
+                  "tcp": {
+                            "port_max": 22,
+                            "port_min": 22
+                        }
+	   },
+	   {
+                  "direction": "inbound",
+                  "name": "allow-ibm-inbound-1",
+                  "source": "0.0.0.0/0",
+                  "icmp": {
+                            "type": 8,
+			    "code": 0
+                        }
+            },
+	    {
+                  "direction": "inbound",
+                  "name": "allow-ibm-inbound-2",
+                  "source": "0.0.0.0/0",
+                     "udp": {
+                            "port_max": 443,
+                            "port_min": 443
+                        }
+            },
+	    {
+                  "direction": "outbound",
+                  "name": "allow-all-outbound",
+                  "source": "0.0.0.0/0",
+                  "tcp": {
+                            "port_max": 22,
+                            "port_min": 22
+                        }
+           },
+	   {
+                  "direction": "outbound",
+                  "name": "allow-ibm-outbound-1",
+                  "source": "0.0.0.0/0",
+                  "icmp": {
+                            "type": 8,
+			    "code": 0
+                        }
+            },
+	    {
+                  "direction": "outbound",
+                  "name": "allow-all-outbound-2",
```

Comment:
Do we not need inbound rules for 992 and 10443?

Reply:
It didnt work with by adding the rules to inbound. I have tested the telnet & web browser by adding the rules to outbound . As the traffic is going from UDP 443 for client-to-site vpn.